tests/ContainerAwareUnitTestCase.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests;
+
+use PHPUnit\Framework\TestCase;
+require_once __DIR__.'/../app/AppKernel.php';
+
+
+class ContainerAwareUnitTestCase extends TestCase {
```

Comment:
Looks like this is now integrated in Symfony, use [KernelTestCase](http://symfony.com/doc/current/testing/doctrine.html) instead